app/knights/knights.py

```diff
@@ -0,0 +1,52 @@
+class Knight:
+    def __init__(
+            self,
+            name: str,
+            power: int,
+            hp: int,
+            weapon: dict,
+            armor: list,
+            potion: dict
+
+    ) -> None:
+        self.name = name.replace("_", " ")
+        self.hp = hp
+        self.power = power
+        self.weapon = weapon
+        self.armor = armor
+        self.potion = potion
+        self.protection = 0
+
+    def common_settings(self) -> dict:
```

Comment:
Make more meaningful and descriptive naming for this function using verbs.

Reply:
I have implemented this functionality in the file "prepare" (old "knights_prepraration")